tests.py

```diff
@@ -9,7 +9,7 @@ def test_app():
         "2",
     ], f"Wrong result for {keyboard} and PIN = 1"
 
-    actual_value = pin_decoder(keyboard, "123")
+    actual_value = sorted(pin_decoder(keyboard, "123"))
```

Comment:
To simplify the hard-coded values, let's choose a pin of length 1 and another one of length 2.